src/sentry/monitors/types.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, Literal, TypedDict
+
+from typing_extensions import NotRequired
+
+
+class CheckinMessage(TypedDict):
+    # TODO(epurkhiser): We should make this required and ensure the message
+    # produced by relay includes this message type
+    message_type: NotRequired[Literal["check_in"]]
+    payload: str
+    start_time: float
+    project_id: str
+    sdk: str
+
+
+class ClockPulseMessage(TypedDict):
```

Comment:
Will be using this type in both the tasks and consumer, don't want an import cycle so it moves here.

Reply:
don't see it in the tasks yet. future work?

Reply:
Yep https://github.com/getsentry/sentry/pull/54647